File: C#/1653. Minimum Deletions to Make String Balanced.cs
public class Solution {
    public int MinimumDeletions(string s) {
        Stack<char> stk = new Stack<char>();
        int res=0;
        for(int i=0;i<s.Length;i++){
            if(s[i] == 'a'){
                if(stk.Count!=0){
                    stk.Pop();
                    res++;
                }
            }
            else{
               stk.Push(s[i]);
            }
        }
        return res;
    }
}

File: C++/1653. Minimum Deletions to Make String Balanced.cpp
class Solution {
public:
    int minimumDeletions(string s) {
        stack<char> stk;
        int res=0;
        for(int i=0;i<s.size();i++){
            if(s[i] == 'a'){
                if(stk.size()!=0){
                    stk.pop();
                    res++;
                }
            }
            else{
               stk.push(s[i]);
            }
        }
        return res;
    }
};
